GoogleForCreators/web-stories-wp
# Links: add branching support in UI

## Context



## Summary



Adds a toggle to regular two-tap links to choose between a regular outbound link and linking to an existing page.

Note: The option is not displayed for single-tap call-to-action links.

## Relevant Technical Choices



## To-do



## User-facing changes



![Screenshot 2023-02-21 at 17 14 38](https://user-images.githubusercontent.com/841956/220399731-8eb6f5ad-fac1-4e7a-8c1f-f7562fd6310f.png)


## Testing Instructions






This PR can be tested by following these steps:

1. Add a bunch of links to a story
2. Verify that linking to URL and linking to a story page works as expected


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11446
